Remotion/Development/UnitTesting.Compilation/Development.UnitTesting.Compilation.csproj

```diff
@@ -16,5 +16,10 @@
   <ItemGroup>
     <PackageReference Include="System.CodeDom" Version="7.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="build\*.*" Pack="true" PackagePath="build\%(Filename)%(Extension);buildTransitive\%(Filename)%(Extension)" />
```

Comment:
oh, something else was missing?

Reply:
I think that is just an artifact caused by reordering commits